### PR TITLE
Feature/13069 fss UI only display filters where there is more than 1 attribute value

### PR DESCRIPTION
**Pull request by prasad14878**

### CODE REVIEW

File: Tests/FunctionalTests/fss-simplified-search-results.spec.ts [hunk 1]
```diff
@@ -93,77 +93,81 @@ describe('Test Search Result Scenario On Simplified Search Page', () => {
      expect(fileDownloadStatus).toContain("check");    
   })
 
-  it('Verify search results specific batch attributes on filter panel', async () => {
+  it('Verify search results specific batch attributes Not displayed on filter panel', async () => {
```
Comment (crdandoukho) [inline Tests/FunctionalTests/fss-simplified-search-results.spec.ts:96]: Please rename this test to `Verify batch attributes with single value are not displayed on filter panel`
```suggestion
  it('Verify batch attributes with single value are not displayed on filter panel', async () => {
```

**Review by crdandoukho — CHANGES_REQUESTED** [review]

File: Tests/FunctionalTests/fss-simplified-search-results.spec.ts [hunk 2]
```diff
@@ -93,77 +93,81 @@ describe('Test Search Result Scenario On Simplified Search Page', () => {
...
 
     const configuredBatchAttibutes=await page.$$eval(pageObjectsConfig.filterBatchAttributes ,elements => { return elements.map(element => element.textContent) });
-    
-    var match = (batchAttributeKeys.length == configuredBatchAttibutes.length) && batchAttributeKeys.every(function(element, index) {
-      return element === configuredBatchAttibutes[index]; 
-     });
-  
-    expect(match).toBeTruthy();           
+    const filterCount=configuredBatchAttibutes.length;
+    if(filterCount>0)
```
[the remark this answers is not in the document]
Comment (kajalkundu) [reply Tests/FunctionalTests/fss-simplified-search-results.spec.ts]: I have updated the changes. Thanks

File: Tests/FunctionalTests/fss-simplified-search-results.spec.ts (hunk 1 above)
Comment (kajalkundu) [reply Tests/FunctionalTests/fss-simplified-search-results.spec.ts:96]: Resolved. Thanks

File: Tests/FunctionalTests/fss-simplified-search-results.spec.ts [hunk 3]
```diff
@@ -93,77 +93,81 @@ describe('Test Search Result Scenario On Simplified Search Page', () => {
...
     await ExpectAllResultsHaveBatchUserAttValue(page, attributeProductType.value);
 
     const filterSpeficAttributeCount=await GetSpecificAttributeCount(page, attributeProductType.key, attributeProductType.value );
     
-    expect(filterSpeficAttributeCount).toEqual(1);    
+    expect(filterSpeficAttributeCount).toEqual(0);    
     
   })
 
   it('Verify configured batch attributes will be displayed on the filter panel', async () => {
```
[the remark this answers is not in the document]
Comment (kajalkundu) [reply Tests/FunctionalTests/fss-simplified-search-results.spec.ts]: Resolved. Thanks

**Review by crdandoukho — APPROVED** [review]
Looks good!

**Review by bhalcha13813 — APPROVED** [review]
LGTM

**Comment by crdandoukho** [comment]
@prasad14878 @kajalkundu could you please resolve the merge conflicts?

**Pull request merged**